packages/zmarkdown/public/css/md.css

```diff
@@ -0,0 +1,3 @@
+@charset "UTF-8";
```

Comment:
Could you please document how to recreate/update this file and its sourcemap?

Reply:
This is juste a Copy/Paste of the file in `<project-root>/public`

Reply:
At the end in think I will update `<project-root>/public` and remove `<project-root>/packages/zmarkdown/public`

Reply:
All we need is a symlink here. Not sure what the question means. :)

Reply:
Nope ^^